src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { DOMWrapper } from './domWrapper'
 import { VueWrapper } from './vueWrapper'
+import { BaseWrapper } from './baseWrapper'
```

Comment:
`BaseWrapper` is only exported with default and not named.

```suggestion
import BaseWrapper from './baseWrapper'
```